subsys/bluetooth/host/gatt.c

```diff
@@ -96,6 +96,27 @@ static ssize_t read_appearance(struct bt_conn *conn,
 				 sizeof(appearance));
 }
 
+#if defined (CONFIG_BT_GAP_PERIPHERAL_PREF_PARAMS)
+static ssize_t read_ppcp(struct bt_conn *conn, const struct bt_gatt_attr *attr,
+			 void *buf, u16_t len, u16_t offset)
+{
+	struct __packed {
+		uint16_t min_int;
+		uint16_t max_int;
+		uint16_t latency;
+		uint16_t timeout;
+	} ppcp;
```

Comment:
By struct I meant something in gatt.h - it might become convenient if we ever add the client-side implementation to this. Anyway, I suppose it's ok to just have this anonymous struct here and then refactor later.

Reply:
yeah, we don't have this kind of struct defined for any of of standard characteristics

Reply:
we do have it for several characteristic descriptors though (I counted three) where it's used exactly like this for the read callback

Reply:
anyway, if you still don't feel like doing it, I'm fine with adding this at some later point as well ;)